pkg/ipamclaimswebhook/podmutator.go

```diff
@@ -65,6 +65,14 @@ func (a *IPAMClaimsValet) Handle(ctx context.Context, request admission.Request)
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	vmName, hasVMAnnotation := pod.Annotations["kubevirt.io/domain"]
+	if !hasVMAnnotation {
+		log.Info(
+			"does not have the kubevirt VM annotation",
+		)
+		return admission.Allowed("not a VM")
+	}
```

Comment:
this shouldn't happen, right ? 

still, I'm OK to leave ... who knows ....

Reply:
> this shouldn't happen, right ?
> 

Yep, but...

> still, I'm OK to leave ... who knows ....

Reply:
let's keep it... 

We can drop it later.